src/main/scala/io/iohk/praos/App.scala

```diff
@@ -46,15 +48,15 @@ object App extends Logger {
     var genesisHistory: GenesisHistory = GenesisHistoryImpl(
       Map(0 -> Genesis(env.initialStakeDistribution, env.initialNonce))
     )
-    // Run the protocol for 1 epoch
-    (1 to env.epochLength).foreach { _ =>
+    // Run the protocol for 3 epoch
+    (1 to env.epochLength * 3).foreach { _ =>
       val slotInEpoch = slotInEpochCalculator.calculate(env.timeProvider)
       var genesis: Genesis = epochGenesisCalculator.computeGenesisForEpoch(slotInEpoch.epochNumber, genesisHistory).getOrElse(
         throw new InvalidStateException("Can not compute a genesis")
```

Comment:
Maybe we could replace this with `IllegalStateException` which doesn't need any new import?

Reply:
:+1: